src/Frontend/Core/Engine/Block/ModuleExtra.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Frontend\Core\Engine\Block;
+
+use Frontend\Core\Engine\TwigTemplate;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+interface ModuleExtra
```

Comment:
Meuh, I'm not that big a fan of this, I'm more in favor of following Symfony. 
@forkcms/core-contributors what do you think?